StarterProject/pageobjects/featuredProductPage.js

```diff
@@ -0,0 +1,63 @@
+
+import Page from "./page.js";
+
+export default class FeaturedProductPage extends Page {
+
+    constructor() {
+        super();     
+    }
+    
+    getAddToBag() { 
+        let addToBag = "div#add-to-cart-container button#add-to-cart";
+        return browser.element(addToBag); 
+    }
+
+    getSelectedQuantity(){
+        let amountSelectedAlready = "#prodCounter";
+        let value = browser.getValue(amountSelectedAlready);
+        return value;
+    }
+
+    getSizeSelected(){
+        let sizeSelected="fieldset#sizeContainer ul.size-buttons li.label";
+        let value = browser.getValue(sizeSelected);
+        return value;
+    }
+
+    getPriceUserSees(){
+        let priceUserSees = "div#product-information div.money span";
+        let value = browser.getText(priceUserSees);
+
+        if(value.charAt(0)=="$"){
+            value=value.substr(1);
+            return parseInt(value);
```

Comment:
nit pick but for `parseInt` it's safest to include the "radix", which is a fancy way of saying make sure we use base10 (normal) numbers, the code should ideally read `parseInt(value, 10);`